Add equality restriction for readonly change (#100)
ballerina-platform/ballerina-release: release-notes/swan-lake-alpha3-release-note.md

##### Debugger

#### Breaking Changes

> 1. Restrict `==` and `!=` equality operators being used with `readonly` type.

Reviewer: ```suggestion
1. Restrict the `==` and `!=` equality operators being used with the `readonly` type.
```

> 1. Restrict the `==` and `!=` equality operators being used with the `readonly` type.

Reviewer 2: How about
```suggestion
1. `==` and `!=` equality expressions can no longer be used with variables of type `readonly`.
```
